packages/dev/gui/src/2D/math2D.ts

```diff
@@ -241,3 +241,20 @@ export class Matrix2D {
         }
     }
 }
+
+export class MathTools {
+    /**
+     * Default rounding precision for GUI elements
+     */
+    public static readonly DefaultRoundingPrecision = 100;

```

Comment:
Maybe it could be interesting that this property is not readonly? This way, the user can change the default precision accross the whole editor easily.

Also, I think the doc should explain that 100 means 2 digits precision, 1000 3 digits and so on, else it may not be obvious for the users.

Reply:
Good points!